.github/dependabot.yml

```diff
@@ -8,8 +8,7 @@ updates:
       separator: '-'
     open-pull-requests-limit: 10
     reviewers:
-      - "haworku"
-      - "ahobson"
```

Comment:
Question: Curious why you removed @haworku and @ahobson ? Maybe we can keep one of them at least in addition to me? Also I'm not familiar with dependabot configuration, is this just who is allowed to approve dependabot PRs? If so I'd add all the people listed as code owners in the `CODEOWNERS` file.

Reply:
I don't know how dependabot works either anymore. We plan to migrate to renovatebot soon anyway. With that in mind this PR is probably premature.